src/docs/src/documentation/content/xdocs/zookeeperReconfig.xml

```diff
@@ -338,7 +338,10 @@ server.3=125.23.63.25:2782:2785:participant</programlisting>
         environment (i.e. behind company firewall). For those users who want to use reconfiguration feature but
         don't want the overhead of configuring an explicit list of authorized user for reconfig access checks,
         they can set <ulink url="zookeeperAdmin.html#sc_authOptions">"skipACL"</ulink> to "yes" which will
-        skip ACL check and allow any user to reconfigure cluster.
+        skip ACL check and allow any user to reconfigure cluster. A more secure mechanism is also provided.
```

Comment:
I removed it - however, the sentence was referring to the "offline" superuser approach as being more secure.